Designer.xaml.cs

```diff
@@ -210,6 +224,27 @@ private void SendToBackButton_Clicked(object? sender, EventArgs e)
         }
     }
 
+    private void DeleteElement(object? sender, EventArgs e)
+    {
+        if (focusedView != null)
+        {
+            // Remove the focused view from its parent layout
+            (focusedView.Parent as Layout)?.Remove(focusedView);
+    
+            // Remove the focused view from the views dictionary and non-tappable views list if necessary
+            views.Remove(focusedView.Id);
+            if (nonTappableTypes.Contains(focusedView.GetType()))
+            {
+                nonTappableViews.Remove(focusedView);
+            }
+    
+            // Clear the focused view and hide the properties frame
+            focusedView = null;
+            PropertiesFrame.IsVisible = false;
```

Comment:
We have had an issue with the property menu not getting visible even when we are next time setting it back to true. Please remove this line.
Also please resolve conflicts and resubmit.